apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/WriteTests.cs

```diff
@@ -0,0 +1,183 @@
+﻿#region
+
+using System;
+using System.Data.Common;
+using System.Threading.Tasks;
+using Xunit;
+
+#endregion
+
+// ReSharper disable RedundantArgumentDefaultValue
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests
+{
+    public class WriteTests : IClassFixture<TestDatabaseFixture>
+    {
+        public WriteTests(TestDatabaseFixture testFixture)
+        {
+            _testFixture = testFixture;
+        }
+
+        private readonly TestDatabaseFixture _testFixture;
+        private string _lastKey;
+
+        private static string UniqueString()
+        {
+            return Guid.NewGuid().ToString();
```

Comment:
whats our coding standards say here?  that we use expression bodies for single lines?
I'm fine changing it here, just want to understand the set of standards (and do we have a doc?)